plugins/terminal.koplugin/main.lua

```diff
@@ -69,29 +414,61 @@ function Terminal:execute()
     end
     UIManager:close(wait_msg)
     local viewer
-    viewer = TextViewer:new {
-        title = _("Command output"),
-        text = table.concat(entries, "\n"),
-        justified = false,
-        text_face = Font:getFace("smallinfont"),
-        -- support re-invoking the terminal with a button:
+    local buttons_table
+    local back_button = {
+        text = _("Back"),
+        callback = function()
+            UIManager:close(viewer)
+            if self.source == "terminal" then
+                self:terminal()
+            else
+                self:manageShortcuts()
+            end
+        end,
+    }
+    local close_button = {
+        text = _("Close"),
+        callback = function()
+            UIManager:close(viewer)
+        end,
+    }
+    if self.source == "terminal" then
         buttons_table = {
             {
+                back_button,
                 {
-                    text = _("Back"),
+                    text = _("Shortcuts"),
+                    -- switch to shortcuts:
                     callback = function()
                         UIManager:close(viewer)
-                        self:onTerminalStart()
+                        self:manageShortcuts()
                     end,
                 },
+                close_button,
+            },
+        }
+    else
+        buttons_table = {
+            {
+                back_button,
                 {
-                    text = _("Close"),
+                    text = _("Terminal"),
+                    -- switch to terminal:
                     callback = function()
                         UIManager:close(viewer)
+                        self:terminal()
                     end,
                 },
+                close_button,
             },
-        },
+        }
+    end
+    viewer = TextViewer:new{
+        title = _("Command output"),
```

Comment:
E.g., I recall we were still discussing changing something like "output" for "results," not in #6598 either, and I don't recall if we even resolved that.

Reply:
I thought someone else would suggest another term, so I've done nothing with this.

Reply:
Is this the exact same phrase as used elsewhere?

Reply:
And where was the discussion?

Reply:
Oh, never mind. This isn't new, just moved.